status_history.go

```diff
@@ -0,0 +1,21 @@
+package braintree
+
+import (
+	"time"
+)
+
+// StatusHistory contains information about the last 50 timestamps where
+// something changed about the subscription.
+type StatusHistory struct {
+	StatusEvents []StatusEvent `xml:"status-event"`
+}
+
+// StatusEvent contains information about what and when something
+// changed about the subscription.
+type StatusEvent struct {
+	Timestamp          time.Time `xml:"timestamp"`
+	Balance            *Decimal  `xml:"balance"`
+	Price              *Decimal  `xml:"price"`
+	Status             string    `xml:"status"`
```

Comment:
Could you make this a type, like `TransactionStatus` is a type?

https://github.com/lionelbarrow/braintree-go/blob/5ae1956a21cedc7ac4a4fe0b5234d2f72c3f0733/transaction.go#L10-L27